php/libraries/Site.class.inc

```diff
@@ -25,6 +26,20 @@ class Site
 {
     var $_siteInfo;
 
+    /**
+     * A short name for the site. E.g. MTL for Montreal.
+     *
+     * @var SiteAlias
+     */
+    var $alias;
+
+    /**
+     * A short name for the MRI.
+     *
+     * @var MRIAlias
+     */
+    var $MRIAlias;
```

Comment:
do we actually use/need this? The difference between a "Site Alias" and "MRI alias" has always been confusing and it doesn't seem like you're updating anything to use it in the PR.

Reply:
I'm not sure. I saw references to it and thought I'd include a value object for it.